test/unit/specs/components/governance/__snapshots__/PageProposal.spec.js.snap

```diff
@@ -216,114 +222,153 @@ exports[`PageProposal has the expected html structure 1`] = `
             <!---->
              
             <!---->
+             
+            <!---->
           </div>
         </div>
+      </div>
+       
+      <div
+        class="row"
+      >
+        <dl
+          class="info_dl info_dl colored_dl"
+        >
+          <dt>
+            Submitted
+          </dt>
+           
+          <dd>
+            a few seconds ago
+          </dd>
+        </dl>
          
-        <div
-          class="row description"
+        <dl
+          class="info_dl colored_dl"
         >
-          <p>
+          <dt>
+            Proposal Status
+          </dt>
+           
+          <dd>
             
-            Submitted a few seconds ago.
             Voting started a few seconds ago.
           
-          </p>
-        </div>
+          </dd>
+        </dl>
          
-        <div
-          class="row page-profile__header__data votes"
+        <dl
+          class="info_dl colored_dl"
         >
-          <dl
-            class="colored_dl"
-          >
-            <dt>
-              Deposit
-            </dt>
-             
-            <dd>
-              
-              200 STAKE
+          <dt>
+            Deposit Count
+          </dt>
+           
+          <dd>
             
-            </dd>
-          </dl>
+            200 STAKE
+          
+          </dd>
+        </dl>
+         
+        <dl
+          class="info_dl colored_dl"
+        >
+          <dt>
+            Vote Count
+          </dt>
            
-          <div
-            class="page-profile__header__data__break"
-          />
+          <dd>
+            0
+          </dd>
+        </dl>
+      </div>
+    </div>
+     
+    <div
+      class="page-profile__section"
+    >
+      <div
+        class="row"
+      >
+        <dl
+          class="info_dl colored_dl"
+        >
+          <dt>
+            Yes
+          </dt>
            
-          <dl
-            class="colored_dl"
-          >
-            <dt>
-              Yes
-            </dt>
-             
-            <dd>
-              0 / 0%
-            </dd>
-          </dl>
+          <dd>
+            0 / 0%
+          </dd>
+        </dl>
+         
+        <dl
+          class="info_dl colored_dl"
+        >
+          <dt>
+            No
+          </dt>
            
-          <dl
-            class="colored_dl"
-          >
-            <dt>
-              No
-            </dt>
-             
-            <dd>
-              0 / 0%
-            </dd>
-          </dl>
+          <dd>
+            0 / 0%
+          </dd>
+        </dl>
+         
+        <dl
+          class="info_dl colored_dl"
+        >
+          <dt>
+            No with Veto
+          </dt>
            
-          <dl
-            class="colored_dl"
-          >
-            <dt>
-              No with Veto
-            </dt>
-             
-            <dd>
-              0 / 0%
-            </dd>
-          </dl>
+          <dd>
+            0 / 0%
```

Comment:
Different PR: We should show some votes in a snapshot to test if displaying the votes works.

Reply:
done.